boards/riscv/esp32c3_devkitm/esp32c3_devkitm_defconfig

```diff
@@ -14,3 +14,5 @@ CONFIG_PINMUX_ESP32=y
 CONFIG_GPIO=y
 CONFIG_GPIO_ESP32=y
 CONFIG_CLOCK_CONTROL=y
+CONFIG_COUNTER=y
+CONFIG_COUNTER_ESP32=y
```

Comment:
I think you can remove this from every esp32 soc as this shall be set automatically from now on.

Reply:
I removed then, thanks @sylvioalves